keysign/gpgmeh.py

```diff
@@ -512,11 +502,6 @@ def sign_keydata_and_encrypt(keydata, error_cb=None, homedir=None):
                     )
 
                 uid_data = UIDExport(signed_keydata, i)
-                # FIXME: Check whether this bug is resolved and the remove this conditional
-                # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=884900
-                if not crashing_gpgme:
-                    log.debug("Data for uid %d: %r, sigs: %r %r", i, uid, uid.signatures, uid_data)
```

Comment:
If we remove this debug line it should be definitely fine.

Reply:
yeah, I didn't intend to remove that line.